playground/index.js

```diff
@@ -1 +1,2 @@
+import "setimmediate";
 require('./src/index');
```

Comment:
Either:
```js
require('setImmediate');
require('./src/index');
```

OR:
```js
import ...
import
```

Please don't mix both styles 😄

And anyways, shouldn't setImmediate be required inside `src/index.js`?